### ISSUE TITLE
Add under maintenance routing to game

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etflex#496]
With recent changes to heat demand in the built environment, we are postponing the update of the game until we have a clear long-term vision for it. This requires both internal and external discussions. In the meantime, the game should be unavailable to our users. To be precise, there are two games, both hosted on https://light.energytransitionmodel.com/. The "Balance energy demand and supply" game already no longer works.

- [ ] Draft a plan to determine what the best way is to prevent users from accessing the game: perhaps https://light.energytransitionmodel.com/ should show a message that it is under maintenance indefinitely, any variations on links to the games should also redirect to this maintenance page. @thomas-qah 
- [ ] Review plan @mabijkerk and @noracato
- [ ] Implement plan @thomas-qah 

<img width="1066" alt="Screenshot 2024-01-18 at 10 46 44" src="https://github.com/quintel/etflex/assets/67338510/e0d12105-d865-4a8c-99ed-37540c1ddb94">

**Comment by thomas-qah** [comment]
Having looked at the structure of the application, I would recommend one of the following actions (plans) to make the game(s) unavailable for now:
1. Use the current application setup and simply do not list any of the games (which in the application are called `scenes`) any longer but replace them with a "under maintenance" message. Scenes are defined (here)[https://github.com/quintel/etflex/blob/master/app/controllers/pages_controller.rb#L18] in the controller, and displayed (here)[https://github.com/quintel/etflex/blob/master/app/views/pages/root.html.haml#L24] in the view.
2. From a security perspective, less complexity equal less attack surface. We could also replace the entire application with a simple static HTML page (maybe two versions, English and Dutch, with the former being the default one) which bypasses the entire Ruby on Rails application.

**Comment by thomas-qah** [comment]
> as long as we do not delete any of the existing content.
Definitely! In option 2 (and option 3, see below), the etflex repository would remain as-is.

> For the local versus global tool we did something similar.
Interesting! That could be option 3 here: simply let requests to `light.energytransitionmodel.com` redirect to a URL in etmodel, such as `energytransitionmodel.com/game` (or `/light` or so), and then display a simple 'under maintenance' page there. This might be the cleanest option. What do you think @mabijkerk?

**Comment by noracato** [comment]
> That could be our third option: simply let requests to light.energytransitionmodel.com redirect to a URL in etmodel (such as energytransitionmodel.com/game

Great thought! I like this solution, simple and clean

**Comment by noracato** [comment]
- [ ] Links quintel.com
- [ ] Fix routing naar juiste url

**Comment by mabijkerk** [comment]
@noracato when I go to https://light.energytransitionmodel.com/ now, I just get "server not found", when I go to https://energytransitionmodel.com/local-global I get "We're sorry, but something went wrong."

Did we remove the server for the first one? Doesn't it still make sense to have some kind of error routing for local-global?

More in general, can you write down why you're closing an issue (such as this one)?
